x/leverage/keeper/oracle.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/umee-network/umee/x/leverage/types"
+)
+
```

Comment:
These methods are all tested via the keeper tests right? We probably should have direct and small tests `oracle_test.go` that test just these. Can we add those? We can update them later once we have a real oracle

Reply:
Small tests have been added in `keeper_test.go` for now. They are also used indirectly during the liquidate tests.

If I separate them into `oracle_test.go`,  should they be written to avoid being methods of `(s *IntegrationTestSuite)`? That was possible with `math_test.go`, for example.